docs/getting_started/installation.md

```diff
@@ -254,6 +254,7 @@ Below you'll find examples showing what the input file might contain for a [Cata
     },
     "server_config": {
         "advanced_settings": {
+            "checksum_read_buffer_size_in_bytes": 1048576,
```

Comment:
noticing lack of

        "agent_factory_watcher_sleep_time_in_seconds": 5,

here in alpha order... need to track that down and perhaps make a new issue...

and in the next section as well...

Reply:
I opened an issue on the discrepancies.

Reply:
https://github.com/irods/irods/issues/7969